bindings/python/test/geometry/d2/objects/test_composite.py

```diff
@@ -0,0 +1,176 @@
+################################################################################################################################################################
+
+# @project        Open Space Toolkit ▸ Mathematics
+# @file           bindings/python/test/geometry/d2/objects/test_composite.py
+# @author         Remy Derollez <remy@loftorbital.com>
+# @license        Apache License 2.0
+
+################################################################################################################################################################
+
+import pytest
+
+import ostk.mathematics as mathematics
+
+################################################################################################################################################################
+
+Object = mathematics.geometry.d2.Object
+Point = mathematics.geometry.d2.objects.Point
+Polygon = mathematics.geometry.d2.objects.Polygon
+Composite = mathematics.geometry.d2.objects.Composite
+Segment = mathematics.geometry.d2.objects.Segment
+
+################################################################################################################################################################
+
+@pytest.fixture
```

Comment:
These fixtures could be added to a `conftest.py` at `d2` level to avoid multiplying the defs across files?